manim/_config/default.cfg

```diff
@@ -146,3 +147,12 @@ log_timestamps = True
 # Uncomment the following line to manually set the loglevel for ffmpeg. See
 # ffmpeg manpage for accepted values
 loglevel = ERROR
+
+
+# Streaming settings
+# I'm using rtp because I've tried it and it works
+[streaming]
+streaming_protocol = rtp
+streaming_ip = 127.0.0.1
+streaming_port = 2000
```

Comment:
Comment, non-blocking: 2000 seems like a very low default port to me, I'd prefer if we would choose a higher port.

Reply:
I guesstimate my way through programming mate. Higher streaming port we may have

Reply:
Mate I guess my merry way through programming in general. Higher port we may have

Reply:
3b1b -> 3818